### ISSUE TITLE
[a11y] CTA button "word lid" te weinig contrast

### ISSUE THREAD

**Issue by hans2103** [fronteers/website#226]
**Describe the bug**
Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds

```
<a class="banner-join-us__link button button-parentheses" href="/nl/word-lid/">
                Word ook lid
            </a>
```

Homepage

<img width="1049" alt="Scherm­afbeelding 2023-05-13 om 16 02 43" src="https://github.com/fronteers/website/assets/639822/8050be9a-67e0-4144-9f5c-cb513a9a2bee">

**Comment by edwinm** [comment]
Changed text color from white to black
